### ISSUE TITLE
Telemetry Parsing Packet Device Debug Returns String Not Integer

### ISSUE THREAD

**Issue by kb1lqd** [FaradayRF/Faraday-Software#47]
The parsing module returns a string for boot counter when it should be an integer. Due to structure parsing no conversions are needed as integers are returned by default.

```python
dictionaryData = {'BootCounter': str(parsed_packet[0]),
                          'ResetCounter': int(parsed_packet[1]),
                          'BrownoutCounter': parsed_packet[2],
                          'Reset_NMICounter': parsed_packet[3],
                          'PMM_LowCounter': parsed_packet[4],
                          'PMM_HighCounter': parsed_packet[5],
                          'PMM_OVP_LowCounter': parsed_packet[6],
                          'PMM_OVP_HighCounter': parsed_packet[7],
                          'WatchdogTimeoutCounter': parsed_packet[8],
                          'FlashKeyViolationCounter': parsed_packet[9],
                          'FLLUnlockCounter': parsed_packet[10],
                          'PeripheralConfigCounter': parsed_packet[11],
                          'AccessViolationCounter': parsed_packet[12],
```

Should be:

``` python
dictionaryData = {'BootCounter': parsed_packet[0],
                          'ResetCounter': parsed_packet[1],
                          'BrownoutCounter': parsed_packet[2],
                          'Reset_NMICounter': parsed_packet[3],
                          'PMM_LowCounter': parsed_packet[4],
                          'PMM_HighCounter': parsed_packet[5],
                          'PMM_OVP_LowCounter': parsed_packet[6],
                          'PMM_OVP_HighCounter': parsed_packet[7],
                          'WatchdogTimeoutCounter': parsed_packet[8],
                          'FlashKeyViolationCounter': parsed_packet[9],
                          'FLLUnlockCounter': parsed_packet[10],
                          'PeripheralConfigCounter': parsed_packet[11],
                          'AccessViolationCounter': parsed_packet[12],
```

**Comment by kb1lqd** [comment]
Committed in branch: https://github.com/FaradayRF/Faraday-Software/commit/e578ba84dfdd16333338f99a3d4761426fef7ef0

**Comment by kb1lqd** [comment]
Created full branch to just fix this bug: 1c27050c4a0bef182f4e8b3cc1d1b839a8009366

**Comment by kb1lqd** [comment]
Fixed in pull commit dad179864cdc8e93b534a4b69039b4a8efd9dcab
